ui/app/src/models/GlobalState.ts

```diff
@@ -63,6 +64,7 @@ export interface GlobalState {
     tools: Array<any>;
     hostSize: WidthAndHeight;
     guest: GuestData;
+    audiencesPanel: AudiencesPanelConfig;
```

Comment:
This is just the config. You may need additional state on the panel so maybe this should just be a prop inside?

e.g.
```ts
audiencesPanel: {
  config: AudiencesPanelConfig;
  currentPersona: ...;
  ...
}
```